proxy_wrapper.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+import os
+import sys
+from arize.otel import register
+from openinference.instrumentation.litellm import LiteLLMInstrumentor
+
+def setup_arize():
+    space_id = os.getenv("ARIZE_SPACE_ID")
```

Comment:
This container will likely be running in Google Cloud Run. Make it so that the Arize keys are read from secret manager with a fallback to os environment variables.

The names of the keys should be `ARIZE_TEST_SPACE_ID` and `ARIZE_TEST_API_KEY`.